e2e-tests/utils.sh

```diff
@@ -108,12 +113,37 @@ function retry_times() {
     return 1
 }
 
+function check_operator_log_for_errors {
+  echo "Checking for operator log errors..."
+  operator_pod_name=$(get_operator_pod_name)
+  errors=$(kubectl logs "${operator_pod_name}" \
+      | grep -v "JmDeploymentStatus.ERROR.Count" \
```

Comment:
Please add a JIRA ticket for all these exceptions we are planning to eliminate. cc @gyfora